src/components/js/EcAccountForm.js

```diff
@@ -187,11 +188,28 @@ export default {
 
   mounted () {
     const $inputs = this.$el.querySelectorAll('input')
+    const sessionCustomer = JSON.parse(sessionStorage.getItem('localCustomer'))
+    const localCustomerKeys = Object.keys(this.localCustomer)
     for (let i = 0; i < $inputs.length; i++) {
       if (!$inputs[i].value) {
         $inputs[i].focus()
         break
       }
     }
+
+    if(sessionCustomer){
+      localCustomerKeys.map(item=>{
+        if (
+            (`${typeof this.localCustomer[item]}` === 'object' && Object.keys(this.localCustomer[item]).length === 0) || 
```

Comment:
```suggestion
            (typeof this.localCustomer[item] === 'object' && Object.keys(this.localCustomer[item]).length === 0) || 
```

Algum motivo pra encapsular o typeof em uma string?

Reply:
Só garantia de retornar uma string mesmo.. as vezes não confio no javascript kk